include/swift/ABI/ObjectFile.h

```diff
@@ -28,10 +28,20 @@ class SwiftObjectFileFormat {
 public:
   virtual ~SwiftObjectFileFormat() {}
   virtual llvm::StringRef getSectionName(ReflectionSectionKind section) = 0;
+  virtual bool sectionContainsReflectionData(llvm::StringRef sectionName) {
```

Comment:
Can you add a Doxygen comment to the function?

Reply:
done